includes/settings/settings.php

```diff
@@ -334,6 +334,18 @@ protected function create_tabs() {
 									'desc' => esc_html__( 'Please note! Allowing uploads of any files (SVG & JSON included) is a potential security risk.', 'elementor' ) . '<br>' . esc_html__( 'Elementor will try to sanitize the unfiltered files, removing potential malicious code and scripts.', 'elementor' ) . '<br>' . esc_html__( 'We recommend you only enable this feature if you understand the security risks involved.', 'elementor' ),
 								],
 							],
+							'google_font' => [
+								'label' => esc_html__( 'Google Fonts', 'elementor' ),
+								'field_args' => [
+									'type' => 'select',
+									'std' => '1',
+									'options' => [
+										'1' => esc_html__( 'Enable', 'elementor' ),
+										'' => esc_html__( 'Disable', 'elementor' ),
+									],
+									'desc' => esc_html__( 'Disable this option if you want to use Google Fonts from a different plugin or theme.', 'elementor' ),
```

Comment:
```suggestion
									'desc' => esc_html__( 'Disable this option if you want to prevent Elementor from loading Google Fonts. This setting is recommended when loading fonts from a different plugin or theme.', 'elementor' ),
```

Reply:
@KingYes 

`Disable this option if you want to prevent Google Fonts from being loaded. This setting is recommended when loading fonts from a different source (plugin, theme or %1$scustom fonts%1$s). `

Reply:
```Disable this option if you want to prevent Google Fonts from being loaded. This setting is recommended when loading fonts from a different source (plugin, theme or %1$sCustom fonts%1$s).```

Target URL: `/wp-admin/admin.php?page=elementor_custom_fonts`

@KingYes